serving/samples/helloworld-nodejs/README.md

```diff
@@ -92,17 +92,13 @@ recreate the source files from this folder.
    # Create and change to the app directory.
    WORKDIR /usr/src/app
 
-   # Copy application dependency manifests to the container image.
-   # A wildcard is used to ensure both package.json AND package-lock.json are copied.
-   # Copying this separately prevents re-running npm install on every code change.
-   COPY package*.json ./
-
-   # Install production dependencies.
-   RUN npm install --only=production
-
    # Copy local code to the container image.
    COPY . .
 
+   # Install production dependencies but discard the node_modules 
+   # copied from your dev environment
+   RUN npm ci --only=production
```

Comment:
The problem I see by forcing to use `npm ci` is that it would fail if there is no `package-lock.json`. This mmight not be desired for a Hello world, since users might just want to copy paste package-lock.json and deploy.

Maybe use an explicit `rm` command  to get riud of `node_modules` and do `npm install`?

Reply:
good catch